admin/create-theme/theme-styles.php

```diff
@@ -29,10 +30,17 @@ public static function build_child_style_css( $theme ) {
 Version: 0.0.1
 License: GNU General Public License v2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Template: {$template}
-Text Domain: {$text_domain}
+";
+
+		if ( ! empty( $template ) ) {
+			$style_css .= "Template: {$template}";
```

Comment:
I tried creating a child theme, and the output is missing a line break:

<img width="361" alt="Screenshot 2023-04-14 at 12 44 33 PM" src="https://user-images.githubusercontent.com/5375500/232106637-01a26778-35fe-4861-bf4d-e030dd7f3227.png">

Should we add one?

```suggestion
			$style_css .= "Template: {$template}\n";
```

Reply:
Thanks for the catch!